### PR TITLE
build: update bci-base image from 15.4 to 15.5

### PR DESCRIPTION
**Pull request by nitendra-suse**
As part of https://github.com/longhorn/longhorn/issues/6206

### CODE REVIEW

**Comment by innobead** [comment]
@nitendra-suse need to sign your commit, or else DCO check will fail.

File: Dockerfile.dapper [hunk 1]
```diff
@@ -16,6 +16,7 @@ ENV DAPPER_SOURCE /go/src/github.com/longhorn/longhorn-engine
 WORKDIR ${DAPPER_SOURCE}
 
 RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:tools:scm/15.5/devel:tools:scm.repo && \
```
Comment (innobead) [inline Dockerfile.dapper:19]: Do you know why we need this repo? is any package unable to install w/o this repo?

Comment (nitendra-suse) [reply Dockerfile.dapper:19]: This was part of migration process for 15.3 to 15.4. It's building with/without these. Let me remove this and update PR.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@mergify backport v1.5.x v1.4.x

**Pull request merged**